secrets/helper/helper.go

```diff
@@ -289,10 +287,8 @@ func InitCloudSecretsManager(secretsConfig *secrets.SecretsManagerConfig) (secre
 
 // MakeKOSKSignature creates KOSK signature which prevents rogue attack
 func MakeKOSKSignature(
-	privateKey *bls.PrivateKey, address types.Address, chainID int64, domain []byte) (*bls.Signature, error) {
-	message, err := abi.Encode(
-		[]interface{}{address, big.NewInt(chainID)},
-		abi.MustNewType("tuple(address, uint256)"))
+	privateKey *bls.PrivateKey, address types.Address, domain []byte) (*bls.Signature, error) {
+	message, err := abi.MustNewType("address").Encode(address)
```

Comment:
Optionally consider to extract `abi.MustNewType("address")` to a package-level variable.

Reply:
Fix: https://github.com/0xPolygon/polygon-edge/pull/1446/commits/5baa9f550555c823722fe79c62f2c9d3077c8954